docs/02-guides/extend-vite-configuration/tailwind.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Tailwind
+description:
+  This guide explains how to setup Tailwind in your Front-Commerce application
+  thanks to Vite.
+---
+
+<p>{frontMatter.description}</p>
+
+:::tip
+
+You can check the `example-extension/tailwind` folder from the skeleton to check
+out it's usage.
+
+:::
+
+## Install Tailwind
+
+```sh
+pnpm add -D tailwindcss
+```
+
+## Generate Tailwind configuration
+
+```sh
+pnpm dlx tailwindcss init -p --ts
+```
+
+## Update your tailwind config
+
+Update your `tailwind.config.ts` to include files from `app/theme`:
+
+```diff
+import type { Config } from 'tailwindcss'
+
+export default {
+- content: []
++ content: ["./app/**/*.{ts,tsx,js,jsx}"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+} satisfies Config
+```
+
+If you have components in some of your extensions, you need to add the related
+extensions' paths to the `content` directive.
+
+## Setup PostCSS
+
+The `tailwindcss init` command should have generated a `postcss.config.js` file,
+you can remove this file and add the `tailwind` configuration to the
+`postcss.config.mjs` file instead:
```

Comment:
Demonstrate the tailwindcss init command.

Remember, this docs goes off of the bases that they have not seen the README.md in the example extension, so I think it should duplicate a lot of those docs.